src/app/config/config.yml

```diff
@@ -14,6 +14,10 @@ framework:
     validation:      { enabled: true, enable_annotations: true, api: auto }
     templating:
         engines: ['twig']
+        assets_base_urls: %assets_base_url%
```

Comment:
There is a problem with the initial call of composer if this parameter has no value. If providing a value overriding it fails though (see bootstrap file below).